main.go

```diff
@@ -226,34 +230,40 @@ func loadPlugins(commander client.Commander, binary *core.NginxBinaryType, env *
 		corePlugins = append(corePlugins, plugins.NewNginxCounter(loadedConfig, binary, env))
 	}
 
-	if (config.AdvancedMetrics{}) != loadedConfig.AdvancedMetrics {
-		corePlugins = append(corePlugins, plugins.NewAdvancedMetrics(env, loadedConfig))
-	}
-
-	if loadedConfig.IsNginxAppProtectConfigured() {
-		napPlugin, err := plugins.NewNginxAppProtect(loadedConfig, env)
-		if err == nil {
-			corePlugins = append(corePlugins, napPlugin)
-		} else {
-			log.Errorf("Unable to load the Nginx App Protect plugin due to the following error: %v", err)
-		}
-	}
-
-	if loadedConfig.NAPMonitoring != (config.NAPMonitoring{}) {
-		nm, err := plugins.NewNAPMonitoring(env, loadedConfig)
-		if err != nil {
-			log.Errorf("Unable to load the Nginx App Protect Monitoring plugin due to the following error: %v", err)
-		} else {
-			corePlugins = append(corePlugins, nm)
+	log.Error(loadedConfig.Extensions)
```

Comment:
Hey, is this supposed to be an Error log ?

Reply:
good spot. I'll remove that.